score-http/score-http/src/main/java/org/oagi/score/gateway/http/helper/Utility.java

```diff
@@ -223,4 +226,15 @@ public static String sha256(String str) {
     public static String emptyToNull(String str) {
         return str.equals("") ? null : str;
     }
+    public static boolean isValidUrl(String url) throws URISyntaxException, MalformedURLException {
+        try{
+            URL urlForTest = new URL(url);
```

Comment:
Should this be 'URI'?

Reply:
URL.toURI() will check the format of URI